test/regression/expected/duckdb_recycle.out

```diff
@@ -0,0 +1,57 @@
+SET duckdb.execution = true;
+CREATE TABLE ta(a INT);
+EXPLAIN SELECT count(*) FROM ta;
+                         QUERY PLAN                         
+------------------------------------------------------------
+ Custom Scan (DuckDBScan)  (cost=0.00..0.00 rows=0 width=0)
+   DuckDB Execution Plan: 
+ 
+ ┌───────────────────────────┐
+ │    UNGROUPED_AGGREGATE    │
+ │    ────────────────────   │
+ │        Aggregates:        │
+ │        count_star()       │
+ └─────────────┬─────────────┘
+ ┌─────────────┴─────────────┐
+ │     POSTGRES_SEQ_SCAN     │
+ │    ────────────────────   │
+ │         Function:         │
+ │     POSTGRES_SEQ_SCAN     │
+ │                           │
+ │         ~2550 Rows        │
```

Comment:
I'm a bit worried this line with the cardinality is going to flaky. I'd be less worried if you added a `pycheck` test instead in `test_explain.py`.

Reply:
As discussed offline - I wasn't able to reproduce the issue with the pycheck. So keeping these for now